ui/dashboards/src/components/Variables/VariableEditorForm/variable-editor-form-model.ts

```diff
@@ -26,12 +26,14 @@ export function getInitialState(initialVariableDefinition: VariableDefinition) {
       kind: '',
       spec: {},
     },
+    customAllValue: undefined as string | undefined,
```

Comment:
Why the type assertion here? I see it’s the same as capturing_regexp, maybe could share a type between those with a more descriptive name?

Reply:
It's a shorthand syntax for defining the acceptable values for `customAllValue` since we use type inference here.

The alternative way would be:
```tsx
type ListVariableFields = {
   customAllValue: string | undefined
}

const listVariableFields: ListVariableFields = {
   customAllValue: undefined
}
```